Sources/XCTest/XCTestCase.swift

```diff
@@ -26,46 +58,46 @@ extension XCTestCase {
             // TODO: When using the Objective-C runtime, XCTest is able to throw an exception from an assert and then catch it at the frame above the test method. This enables the framework to effectively stop all execution in the current test. There is no such facility in Swift. Until we figure out how to get a compatible behavior, we have decided to hard-code the value of 'true' for continue after failure.
         }
     }
-    
-    public func invokeTest() {
```

Comment:
`-[XCTestCase invokeTest]` is a public method in Apple XCTest (although the Apple documentation does state that "in general this should not be called directly".

Removing this method here isn't necessarily a blocker for merging this pull request, but it is one step backwards in terms of achieving API parity. Anyway just consider this a footnote of mine, as personally I think merging this pull request in order to get a correct `override func setUp()` is more important.